components/progressTracker/ProgressTracker.js

```diff
@@ -0,0 +1,47 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import theme from './theme.css';
+
+import Box from '../box';
+import ProgressStep from './ProgressStep';
+
+class ProgressTracker extends PureComponent {
+  render() {
+    const { color, children, activeStep } = this.props;
+
+    const classNames = cx(theme['progress-tracker']);
+
+    return (
+      <Box className={classNames}>
+        {React.Children.map(children, (child, index) => {
+          const currentActiveStep = activeStep < 0 ? 0 : activeStep;
```

Comment:
Q: How exactly can currentActiveStep ever be `< 0`? The default value is 0 so that shouldn't be a problem right?

Reply:
Okay I guess it would be a safety, although it would also be a sign for the props being passed that the activeStep is `< 0` and therefor incorrect. Now if you would pass something that's smaller than 0 it will just put the step to 0 and it won't look as if something's wrong. Which is in a way a less destructive way if anything were to go wrong... I can live with the check

 :ok: by @InstaK on 18/06/2018 13:54:36